README.md

```diff
@@ -138,13 +138,13 @@ class Address extends ViewableData
 }
 ```
 
-A great example of this is SilverStripe's `Image` class. When you call `$MyImage` on a template, it invokes its `forTemplate()` method, which returns a string of HTML representing an `<img />` tag with all the correct attributes and values.
+A great example of this is SilverStripe's `Image` class. When you call `$MyImage` on a template, it invokes `Image::forTemplate()` method, which returns a string of HTML representing an `<img />` tag with all the correct attributes and values.
```

Comment:
If this change is to stay it probably needs to read "it invokes _the_ ... method"

Reply:
I could swear I remember editing that in, but clearly not. Perhaps it was elsewhere, heh.
Probably couldn't make up my mind between "the method" or just simply naming the signature.